lib/rules/require-spaces-around-generators.js

```diff
@@ -78,8 +78,6 @@ module.exports.prototype = {
                 return;
             }
 
-            // for a named function, use node.id
-            var functionNode = node.id || node;
```

Comment:
basically don't want to use `node.id` since that makes the first token start and the function name